tests/serialized_test_data_generation/make_rundir.py

```diff
@@ -0,0 +1,19 @@
+import fv3config
+import yaml
+import shutil
+import os
+import sys
+import subprocess
+
+experiment = sys.argv[1]
+workdir = os.getcwd()
+target_directory = workdir + '/rundir/' + experiment
+os.makedirs(target_directory, exist_ok=True)
+shutil.copy("submit_job.sh", target_directory)
+config = yaml.safe_load(open(os.path.join(workdir + "/configs",experiment + ".yml"), "r"))
+layout = config["namelist"]["fv_core_nml"]["layout"]
+num_ranks = 6 * layout[0] * layout[1]
+subprocess.call(["sed -i '' 's/NUM_RANKS\=6/NUM_RANKS\=" + str(num_ranks) + "/g' " + os.path.join(target_directory, "submit_job.sh")], shell=True)
```

Comment:
This does feel a bit weird. I think Jeremy's suggestion could make this script a lot easier to follow (or I guess make it superfluous).

Reply:
Thanks! Sure I'll give it a try -- I am not a big fan of using sed, so that sounds better